### ISSUE TITLE
howdoi doesnt work

### ISSUE THREAD

**Issue by sonuishaq67** [gleitz/howdoi#351]
![Screenshot from 2021-03-01 19-59-16](https://user-images.githubusercontent.com/55189255/109511012-a7ab4780-7ac8-11eb-8edb-fbe9d28449e0.png)

error log
https://pastebin.ubuntu.com/p/gtKB28Br3N/

**Comment by sonuishaq67** [comment]
OS arch linux 
python version : Python 3.9.2

**Comment by gleitz** [comment]
@Srinivasa314 was having the same issue, so consolidating the bug reports here.

**Comment by gleitz** [comment]
@sonuishaq67 can you confirm if this error happens for every query, or just some queries?

Could you also try clearing your cache with the `-C` flag?

While we're debugging this issue, I think you should be able to use bing or duckduckgo to fetch results by changing the search engine with `-e bing`.

cc @SaurusXI

**Comment by sonuishaq67** [comment]
> -e bing , duckduckgo

it works for bing and duckduckgo 

> can you confirm if this error happens for every query, or just some queries?

happens for every query with google

> -C flag

I cleared the cache but its the same for google

**Comment by sonuishaq67** [comment]
@gleitz  It's working now when I clear the cache

**Comment by sonuishaq67** [comment]
It works sometimes when I clear the cache and sometimes it doesn't

**Comment by gleitz** [comment]
If you can share other queries and error responses that would be useful. We're trying to track down that the issue with the URL might be.

**Comment by sonuishaq67** [comment]
I hope this helps 
https://pastebin.ubuntu.com/p/5FSnZQDyVH/

**Comment by gleitz** [comment]
We have a fix checked into the master branch that I believe should fix this issue. Would you mind checking out howdoi and testing it for us?

```
git clone git@github.com:gleitz/howdoi.git
cd howdoi
pip3 install -r requirements.txt
```

Then try a few queries using this format

```
python -m howdoi.howdoi print in c -C
```

And then let me know if you run into any issues. Really appreciate the help!

**Comment by sonuishaq67** [comment]
Yep this works perfectly :) 
Thanks for fixing.

Will be closing this.

**Comment by gleitz** [comment]
Fixed in 2.0.14
